### ISSUE TITLE
script to reproduce cifar expr in table 5

### ISSUE THREAD

**Issue by yuanzhi-zhu** [atong01/conditional-flow-matching#38]
Massive appreciation for the remarkable work on this repository!!

I would like to know how to reproduce the results in table 5, in your paper _Improving and Generalizing Flow-Based
Generative Models with Minibatch Optimal Transport_.

I am a bit confused as it looks like the readme has to be updated to be consistent with the v1 codebase. 

Thank you in advance!

**Comment by atong01** [comment]
Yes indeed! The Readme needs to be updated in this respect. We are still working on making all experiments nicely reproducible. Will open a PR soon to sort this but in short, you should run

```
python src/train.py -m experiment=image_otcfm,image_fm,image_cfm
```
to run all three at once / in series depending on your compute setup.

to test these models once trained I used

```
python src/eval.py -m ckpt_path=XXXX ++model.test_nfe=1,5,10,100
```

all from the `runner` directory. This will of course depend on your compute setup, but this defaults to DDP on 2 GPUs.
You may also need to point to your own CIFAR-10 saved file for the FID computation as I believe I hardcoded the path in.

Let me know if you run into any issues!

**Comment by kilianFatras** [comment]
Hi, 

We have added a first cifar10 code. It should reproduce our experiments and we will improve it soon. https://github.com/atong01/conditional-flow-matching/tree/main/examples/cifar10.
